airgun/widgets.py

```diff
@@ -1067,9 +1067,9 @@ class Pf4ConfirmationDialog(ConfirmationDialog):
     right corner."""
 
     ROOT = '//div[@id="app-confirm-modal" or @data-ouia-component-type="PF4/ModalContent"]'
-    confirm_dialog = PF4Button('OUIA-Generated-Button-danger-1')
-    cancel_dialog = PF4Button('OUIA-Generated-Button-link-1')
-    discard_dialog = PF4Button('OUIA-Generated-Button-plain-1')
+    confirm_dialog = PF4Button(locator='//button[text()="Confirm"]')
```

Comment:
Why remove the ouia locators?

Reply:
These locators are not stable and change when you repeat the step. e.g. when you try to delete the entity 2 times the locator id changes. I thought this would be more brittle than that.

Reply:
I think you can address this with the UXD team to get static ids

Reply:
Just checked with the UX team https://github.com/theforeman/foreman/commit/4bcd67ddb14abad4c9b44e213e19e1dfda7c8b7e here is the updated PR land in 6.12